packages/transaction-controller/src/utils/feature-flags.ts

```diff
@@ -34,6 +50,17 @@ export type TransactionControllerFeatureFlags = {
   [FEATURE_FLAG_TRANSACTIONS]?: {
     /** Maximum number of transactions that can be in an external batch. */
     batchSizeLimit?: number;
+
+    /** Fallback gas estimation configurations per chain. */
+    gasEstimateFallbacks?: {
+      [chainId: Hex]: GasEstimateFallback;
+    };
+
+    /**
+     * Default gas estimate fallback.
+     * This value is used when no specific gas estimate fallback is found for a chain ID.
+     */
+    defaultGasEstimateFallback?: GasEstimateFallback;
```

Comment:
Could we combine these two properties into a single object like we do for transaction polling such as:

```
gasEstimateFallback: {
    perChain?: {
        [chainId: Hex]: GasEstimateFallback;
    };
    default?: GasEstimateFallback;
}
```

Reply:
yes, I've done something similar to transaction polling as you suggested.